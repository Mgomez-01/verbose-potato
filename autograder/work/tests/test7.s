    SECTION "code", ROM0
    ; call, ret

    ld sp, $3FF

    ld b, $6
    ld a, $0
    call func
    ld [$400], a

    jp $400
    halt


func:

    add a, b
    ld c, a
    ld a, b
    cp a, 0
    jp z, done
    ld a, c
    dec b
    call func
done:
    ld a, c
    ret
