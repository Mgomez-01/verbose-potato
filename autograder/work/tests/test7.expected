15
TEST EXITED NORMALLY
